README.md

```diff
@@ -241,14 +241,6 @@ e.g.:
 
     Native["java/lang/System.arraycopy.(Ljava/lang/Object;ILjava/lang/Object;II)V" = function(src, srcOffset, dst, dstOffset, length) {...};
 
-If you need to implement a method in JS but you can't declare it `native` in Java, use `Override`.
```

Comment:
Unfortunately in order to fix #1234 and enable optimizations in lazy linking, we need to get rid of overrides.
To hook methods while debugging, we could do:
1) In vm/parser.ts, in MethodInfo::isNative return true for the method we want to hook
2) Create a native in native.js
I think this should work, but supporting it in a release build is going to slow us down. We could support it only on debug builds.
I'm going to try and implement this approach in a follow-up (since in this PR I'm not yet removing overrides).